internal/inspect/policy.go

```diff
@@ -229,8 +238,25 @@ func (pol *Policy) Results(ctx context.Context, loadPolicy loadPolicyFn) (map[st
 							Source: namer.PolicyKeyFromFQN(wrapper.FQN),
 							Used:   true,
 						})
-
 						variables[importedVarName] = true
+
+						referencedAttributes := make(map[string]*responsev1.InspectPoliciesResponse_Attribute)
+						if err := pol.referencedAttributesInExpr(importedVariable.Value, referencedAttributes); err != nil {
+							return fmt.Errorf("failed to find referenced attributes in imported variable: %w", err)
+						}
+
+						if len(referencedAttributes) > 0 {
+							ss := make(util.StringSet)
+							for _, attr := range pol.results[policyID].Attributes {
+								ss[attr.Name] = struct{}{}
+							}
```

Comment:
Isn't this set created over and over for every imported variable that references an attribute? I suppose there won't be that many in practice but there could be degenerate cases.